packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import { DeploymentSummary } from "./utils/DeploymentSummary.sol";
+import { KontrolUtils } from "./utils/KontrolUtils.sol";
+import { Types } from "src/libraries/Types.sol";
+import {
+    IOptimismPortal as OptimismPortal,
+    ISuperchainConfig as SuperchainConfig
+} from "./interfaces/KontrolInterfaces.sol";
+
+contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
+    OptimismPortal optimismPortal;
+    SuperchainConfig superchainConfig;
+
+    function setUp() public {
+        recreateDeployment();
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+    }
+
+    /// TODO: Replace struct parameters and workarounds with the appropriate
+    /// types once Kontrol supports symbolic `bytes` and `bytes[]`
+    function prove_proveWithdrawalTransaction_paused(
+        // WithdrawalTransaction args
+        uint256 _tx0,
+        address _tx1,
+        address _tx2,
+        uint256 _tx3,
+        uint256 _tx4,
+        // bytes   memory _tx5,
+        uint256 _l2OutputIndex,
+        // OutputRootProof args
+        bytes32 _outputRootProof0,
+        bytes32 _outputRootProof1,
+        bytes32 _outputRootProof2,
+        bytes32 _outputRootProof3
+    )
+        external
+    {
+        bytes memory _tx5 = freshBigBytes(320);
+
+        bytes[] memory _withdrawalProof = freshWithdrawalProof();
+
+        Types.WithdrawalTransaction memory _tx = createWithdrawalTransaction(_tx0, _tx1, _tx2, _tx3, _tx4, _tx5);
+        Types.OutputRootProof memory _outputRootProof =
+            Types.OutputRootProof(_outputRootProof0, _outputRootProof1, _outputRootProof2, _outputRootProof3);
+
+        // After deployment, Optimism portal is enabled
+        require(optimismPortal.paused() == false, "Portal should not be paused");
+
+        // Pause Optimism Portal
+        vm.prank(optimismPortal.GUARDIAN());
+        superchainConfig.pause("identifier");
```

Comment:
This interface is outdated. 
```suggestion
        superchainConfig.pause();
```

Reply:
We should define the interface in `SuperchainConfig.sol` and then import it in here. We decided on splitting out interfaces in the same file as the impl previously, this will guarantee it doesn't fall out of sync if we add the inheritance

Reply:
The interface is correct here (https://github.com/ethereum-optimism/optimism/pull/8634#discussion_r1450695089) but we did discuss generating separate interface files as a measure to improve compilation times. So I figured for now we'd leave it as-is (a test will fail if the interface does not match), and switch it over once that's done

Reply:
I'd rather not add the interface to `SuperchainConfig.sol` in this PR, because when we do that, we'll also want to change tests and scripts to use that interface instead. Doing that here would bloat/delay this PR, so it makes sense to keep them separate IMO

Reply:
SGTM